docs/TwoHostMultiVlanDemo.md

```diff
@@ -6,91 +6,65 @@ In this example we will simulate a two host cluster (with the two hosts being si
 
     ```
     cd $GOPATH/src/github.com/contiv/netplugin
-    CONTINV_NODES=2 make demo
+    CONTIV_NODES=2 make build demo
     ```
     
-    Note: User may simulate more hosts by setting the CONTINV_NODES variable to a desired number. Each host corresponds to a vagrant-vm, which are connected through their 'eth2' interface to a virtualbox bridge network for the container data traffic.
-    
+    Note: User may simulate more hosts by setting the CONTIV_NODES variable to a desired number. Each host corresponds to a vagrant-vm, which are connected through their 'eth2' interface to a virtualbox bridge network for the container data traffic.
+
 2. Once the make is done, start a separate ssh session to each node and run the netplugin:
 
     ```
-    CONTINV_NODES=2 vagrant ssh netplugin-node1
-    sudo $GOPATH/bin/netplugin -host-label host1
+    CONTIV_NODES=2 vagrant ssh netplugin-node1
+    sudo $GOPATH/bin/netmaster &
+    sudo $GOPATH/bin/netplugin -host-label host1 &
     ```
     ```
-    CONTINV_NODES=2 vagrant ssh netplugin-node2
-    sudo $GOPATH/bin/netplugin -host-label host2
+    CONTIV_NODES=2 vagrant ssh netplugin-node2
+    sudo $GOPATH/bin/netplugin -host-label host2 &
     ```
-3. Let's create 4 containers two on each vagrant node that we will add to the two networks viz. orange and purple later.
 
-    On netplugin-node1:
-    ```
-    sudo docker run -it --name=myContainer1 --hostname=myContainer1 ubuntu /bin/bash
-    sudo docker run -it --name=myContainer2 --hostname=myContainer2 ubuntu /bin/bash
-    ```
-    On netplugin-node2:
-    ```
-    sudo docker run -it --name=myContainer3 --hostname=myContainer1 ubuntu /bin/bash
-    sudo docker run -it --name=myContainer4 --hostname=myContainer2 ubuntu /bin/bash
-    ```
-4. Now let's load the multi-host multi vlan configuration from the [../examples/two_hosts_multiple_vlans_nets.json](../examples/two_hosts_multiple_vlans_nets.json) file by issuing the following commands from one of the vagrant vms.
+3. Load the multi-host multi vlan configuration from the [../examples/two_hosts_multiple_vlans_nets.json](../examples/two_hosts_multiple_vlans_nets.json) file by issuing the following commands from one of the vagrant vms.
 
     ```
-    CONTINV_NODES=2 vagrant ssh netplugin-node1
+    CONTIV_NODES=2 vagrant ssh netplugin-node1
     cd $GOPATH/src/github.com/contiv/netplugin
     netdcli -cfg examples/two_hosts_multiple_vlans_nets.json
     ```
-5. Now everything should be setup as per the diagram and we are good to test the connectivity.
 
-    Determine the IP addresses assigned to the container `myContainer3` and `myContainer4` by running command like `ifconfig` or `ip address show` from the container shells (opened in step 3).
-    Note: the current implementation with ovs names the netdevices as 'port<number>'
+4. Launch 4 containers, two on each vagrant node that we will add to the two networks viz. orange and purple later.

```

Comment:
@mapuri - thanks for catching this. `later` shouldn't have been there. I corrected that in the updated PR